components/data-services/org.wso2.carbon.dataservices.core/pom.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Why the new line is removed? Is there any particular reason?

Reply:
Jacoco plugin was integrated using an automation tool and that tool perform code formatting